sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Generated/ConversationAuthoringClient.cs

```diff
@@ -1155,14 +1155,14 @@ public virtual Pageable<BinaryData> GetTrainedModels(string projectName, Request
         /// </summary>
         /// <param name="projectName"> The name of the project to use. </param>
         /// <param name="trainedModelLabel"> The trained model label. </param>
-        /// <param name="stringIndexType"> Specifies the method used to interpret string offsets. Set this to "Utf16CodeUnit" for .NET strings, which are encoded as UTF-16. Allowed values: "Utf16CodeUnit". </param>
+        /// <param name="stringIndexType"> Specifies the method used to interpret string offsets. Set this to "Utf16CodeUnit" for .NET strings, which are encoded as UTF-16. The default is "Utf16CodeUnit". Allowed values: "Utf16CodeUnit". </param>
```

Comment:
This is incorrect. Not only are you forcing the customer to pass something - could it be null, "", or the "Utf16CodeUnit" value? - but there's only one allowed option and you aren't defaulting anything in code. The code should be left as-is or don't in the message creation method if null or empty, but even that should be default. Basically, there is zero reason a customer using this SDK should ever need to set this. All `String` types are UTF16 in .NET. It doesn't matter that the service only supports this value anyway.

I don't understand why this was even changed in the generator.

Reply:
Hello @heaths We will remove all the client default value (set by x-ms-client-default in swagger) of a parameter in the method signature. And force customer to pass value when call the method.
In above case, `stringIndexType` is an enum with only one enum option, in that case we will set the option `Utf16CodeUnit` as default value. But now we decide to ignore all those default value.

Reply:
This also is a breaking change.

Reply:
@chunyu3 last we discussed you mentioned all affect libraries were not GA, but it does appear that https://www.nuget.org/packages/Azure.AI.Language.Conversations is GA.

We will need to mitigate any GA'd libraries with a backwards compatible change.

Reply:
if this is a required constant I wonder if we should simply remove it from the signature all together since any other value than the one will fail, and only when the service adds a second value introduce it on the signature.  Do we know if this marked as a constant in the swagger?

Reply:
It's a single-valued enum currently, but also modelAsString: true. Compared with related endpoints, there are 2 more possibilities they could likely add in the future, so modelAsString: true is warranted.

That said, in QA (a related library) I defined it as an internal defualt on its ClientOptions, but could one day make that public and let users override that, though there's little reason to. So, in effect, it is a constant for the entire .NET library - for all methods/models that use it.